### PR TITLE
[bitnami/flux] fix: :bug: :lock: Add missing notification controller ports

### PR DESCRIPTION
**Pull request by javsalgar**
Signed-off-by: Javier Salmeron Garcia <jsalmeron@vmware.com>

### Description of the change
This PR adds to the deployment spec the ports that were missing in the notification controller. We also created two extra services following the upstream deployment specs

https://github.com/fluxcd/notification-controller/blob/c6b89cd2d65dc46951b5e7657e0a40d1a1209b28/config/manager/deployment.yaml#L14

### Benefits

Improved security of the application

### Possible drawbacks
n/a

### Applicable issues

- fixes #

### Additional information

Thanks to @jlmartinnavarro

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/flux/values.yaml [hunk 1]
```diff
@@ -1760,11 +1760,15 @@ notificationController:
   ##
   replicaCount: 1
   ## @param notificationController.containerPorts.metrics Notification Controller metrics container port
+  ## @param notificationController.containerPorts.receiver Notification Controller receiver container port
   ## @param notificationController.containerPorts.health Notification Controller health container port
+  ## @param notificationController.containerPorts.webhook Notification Controller receiver container port
```
Comment (dgomezleon) [inline bitnami/flux/values.yaml:1765]:
```suggestion
  ## @param notificationController.containerPorts.webhook Notification Controller webhook container port
```

File: bitnami/flux/values.yaml [hunk 2]
```diff
@@ -2082,6 +2086,114 @@ notificationController:
...
   initContainers: []
+  ## @section Notification Controller Traffic Exposure Parameters
+  service:
+    ## Notification Controller Receiver service parameters
+    ##
+    receiver:
+      ## @param notificationController.service.receiver.type Notification Controller service type
+      ##
+      type: ClusterIP
+      ## @param notificationController.service.receiver.ports.http Notification Controller service metrics port
```
Comment (dgomezleon) [inline bitnami/flux/values.yaml:2097]:
```suggestion
      ## @param notificationController.service.receiver.ports.http Notification Controller service receiver port
```

File: bitnami/flux/values.yaml [hunk 3]
```diff
@@ -2082,6 +2086,114 @@ notificationController:
...
+      ##     timeoutSeconds: 300
+      ##
+      sessionAffinityConfig: {}
+    ## Notification Controller webhook service parameters
+    ##
+    webhook:
+      ## @param notificationController.service.webhook.type Notification Controller service type
+      ##
+      type: ClusterIP
+      ## @param notificationController.service.webhook.ports.http Notification Controller service metrics port
```
Comment (dgomezleon) [inline bitnami/flux/values.yaml:2150]:
```suggestion
      ## @param notificationController.service.webhook.ports.http Notification Controller service webhook port
```

File: bitnami/flux/values.yaml [hunk 4]
```diff
@@ -2082,6 +2086,114 @@ notificationController:
...
+      ##
+      ports:
+        http: 80
```
Comment (dgomezleon) [inline bitnami/flux/values.yaml:2153]: 9292

File: bitnami/flux/values.yaml [hunk 5]
```diff
@@ -2082,6 +2086,114 @@ notificationController:
...
+    ## Notification Controller Receiver service parameters
+    ##
+    receiver:
+      ## @param notificationController.service.receiver.type Notification Controller service type
+      ##
+      type: ClusterIP
+      ## @param notificationController.service.receiver.ports.http Notification Controller service metrics port
+      ##
+      ports:
+        http: 80
```
Comment (dgomezleon) [inline bitnami/flux/values.yaml:2100]: 9090

**Review by dgomezleon — CHANGES_REQUESTED** [review]

File: bitnami/flux/values.yaml (hunk 4 above)
Comment (javsalgar) [reply bitnami/flux/values.yaml:2153]: In principle there's no need to make the service port match the container port. As it is an http port it makes more sense to use the standard 80 port

File: bitnami/flux/values.yaml (hunk 5 above)
Comment (javsalgar) [reply bitnami/flux/values.yaml:2100]: Same reasoning as here https://github.com/bitnami/charts/pull/25075#discussion_r1559106646

**Review by dgomezleon — APPROVED** [review]

**Pull request merged**